spyder/app/tour.py

```diff
@@ -144,68 +144,76 @@ def get_tour(index):
              'interact': False},
             ]
 
-    intro = [{'title': _("Welcome to the Introduction tour"),
+    intro = [{'title': _("Welcome to Spyder!"),

```

Comment:
I just realised that if users go to the tour from the dialog that we just added in PR#13953 they will see "Welcome to Spyder" two times. One on that dialog and the other one in the introduction tour first dialog so maybe we should leave this one as "Welcome to the Introduction tour". What do you think @ccordoba12 and @CAM-Gerlach?

Reply:
That's a good point. What about if instead of repeating `Welcome` twice, we say here instead `Explore what Spyder has to offer` or `Learn about Spyder's cool features`.

Those are just suggestions, but my idea is to mention something more appealing and fun, instead of restating the obvious.

Reply:
Excellent point @juanis2112 . While the current title may not be the most eye-catching, we have to remember that this is the name of the first step displayed in the popup menu, not body text or the title of the whole tour, so it needs to be kept short, appropriate and descriptive of that step.

Therefore, I going with @juanis2112 's suggestion of reverting to the original text (or perhaps a less drastic variation, if either of you have further suggestions:

```suggestion
    intro = [{'title': _("Welcome to the introduction tour!"),
```

Reply:
Ok, no problem if the two of you agree.

Reply:
Yeah I agree with @CAM-Gerlach

Reply:
Okay, thanks. Pushed.